Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -2171,7 +2171,8 @@ static void setDynamicRangeLimit(CALayer* layer, PlatformDynamicRangeLimit platf
 void WebViewImpl::applicationShouldSuppressHDR(bool suppress)
 {
     m_page->setShouldSuppressHDR(suppress);
-    setDynamicRangeLimit(m_rootLayer.get(), suppress ? PlatformDynamicRangeLimit::defaultWhenSuppressingHDR() : PlatformDynamicRangeLimit::noLimit(), true);
+    if (m_page->protectedPreferences()->acceleratedDrawingEnabled())
+        setDynamicRangeLimit(m_rootLayer.get(), suppress ? PlatformDynamicRangeLimit::defaultWhenSuppressingHDR() : PlatformDynamicRangeLimit::noLimit(), true);
```

Comment:
Since this is temporary bug-fixing, and we'll eventually need to get this working properly, would it be possible to file a follow-up to https://bugs.webkit.org/show_bug.cgi?id=291534, and add a `// FIXME` comment here with the new bug number?

Reply:
That is a good idea, though I wonder if we need to support the `!acceleratedDrawingEnabled()` path? It's an internal setting and otherwise always 1 in the UIProcess afaik

Reply:
Ideally we'd want the behavior to stay the same between the two modes, but yeah if that's just an internal flag mostly used for debugging, it could potentially stay unsupported.
That's why I said "optionally" ;-) I'll let you decide.
Thank you for the quick patch.